src/main/java/de/siegmar/securetransfer/controller/ReceiveController.java

```diff
@@ -65,9 +68,13 @@ public ReceiveController(final MessageReceiverService messageService) {
      * Ask for message retrieval (return confirm or password dialog).
      */
     @GetMapping("/{id:[a-f0-9]{64}}")
-    public String receive(@PathVariable("id") final String id, final Model model) {
+    public String receive(@PathVariable("id") final String id,
```

Comment:
With more than one request data (path variable + request param) I'd suggest to introduce a RequestDTO class.